woocommerce-gutenberg-products-block.php

```diff
@@ -110,7 +110,20 @@ function() {
 }
 
 add_action( 'plugins_loaded', array( '\Automattic\WooCommerce\Blocks\Package', 'init' ) );
+add_action( 'plugins_loaded', 'woocommerce_blocks_reinitialize_feature_flag', 11 );
 
+/**
+ * Reinitialize the feature flag.
+ *
+ * We want to redefine the lower feature flag to 2 if we're running from the feature plugin.
+ * This file (and code) will only run in the plugin and not WooCommerce core.
+ *
+ * @since 2.7.0
+ */
+function woocommerce_blocks_reinitialize_feature_flag() {
+	global $woocommerce_blocks_phase;
+	$woocommerce_blocks_phase = '2';
```

Comment:
Incidentally, I understand we're using numeric values here but I think it would be much much more clear to consumers if we used named contexts e.g. WOOCOMMERCE_BLOCKS_CONTEXT = feature_plugin or package?